docs/guides/quick-start.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Quick Start Guide
+title: Quick Start
```

Comment:
I think just Quick Start is more like a standard name along with Installation or Getting Started (you skim for them usually)